spring-kitchensink-controlleradvice/functional-tests/src/test/java/org/jboss/as/quickstarts/kitchensink/test/page/RegistrationFormPageFragment.java

```diff
@@ -118,6 +112,14 @@ public void register(Member member, boolean fillDate) {
             today.click();
             javascript.executeScript("document.getElementById('datepicker').blur()");
         }
+        nameField.click();
```

Comment:
@trepel : Did you intend to add the nameField.click()? I assume that is done to put the cursor in the next field, but just want to be sure. :-)